spec/fortnox/api/repositories/examples/find_examples.rb

```diff
@@ -0,0 +1,25 @@
+shared_examples_for '.find' do |id_attribute|
```

Comment:
If not, we need to convert the attribute name from JSON format (as stated in the repository) to a symbol. I believe this functionality is built into the repository as a private method? Should we make that method public so we can use it here in the test? Or is it simple to write?